packages/flutter/lib/src/material/tabs.dart

```diff
@@ -488,14 +529,16 @@ class TabBar extends StatefulWidget implements PreferredSizeWidget {
     this.indicatorColor,
     this.indicatorWeight: 2.0,
     this.indicatorPadding: EdgeInsets.zero,
+    this.indicator, // TBD indicator
```

Comment:
Should we say something about the nullability of this in the constructor's doc?

Reply:
Yes, I added a sentence about indicatorColor et al being ignored if indicator is specified.